mnist-classifaction/deploy_model/deploy.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import logging
+from servicefoundry import (
+    Build,
+    PythonBuild,
+    Service,
+    Resources,
+    Port,
+    ArtifactsDownload,
+    TruefoundryArtifactSource,
+)
+
+logging.basicConfig(level=logging.INFO)
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--workspace_fqn", type=str, required=True)
+parser.add_argument("--model_version_fqn", type=str, required=True)
+parser.add_argument("--host", type=str, required=True)
+parser.add_argument("--path", type=str, required=False)
+args = parser.parse_args()
+
+service = Service(
+    name="mnist-classification-svc",
+    image=Build(
+        build_spec=PythonBuild(
+            command="python gradio_demo.py",
+            requirements_path="requirements.txt",
+        )
+    ),
+    ports=[Port(port=8000, host=args.host, path=args.path)],
+    resources=Resources(
+        memory_limit=500,
+        memory_request=500,
+        ephemeral_storage_limit=600,
+        ephemeral_storage_request=600,
+        cpu_limit=0.3,
+        cpu_request=0.3,
+    ),
+    artifacts_download=ArtifactsDownload(
+        artifacts=[
+            TruefoundryArtifactSource(
```

Comment:
I am worried that this example will not work if someone uses Civo or GCP.

Reply:
It will still work, right? via init container download?

Reply:
yes - I have disabled volume cache. So it will work everywhere

Reply:
> I have disabled volume cache.

Right, I missed this.